frontend/src/app/spot/components/selector-field/stories/SelectorFieldFontWeight.example.html

```diff
@@ -0,0 +1,12 @@
+<div class="spot-container">
+  <spot-selector-field
+    [label]="'Bold Label'"
+    [labelWeight]="'bold'"
```

Comment:
Same here

Reply:
For the label I agree, it is a mere string that gets rendered. Here I have a different opinion. two reasons:

- bindings give you syntax highlighting in IDEs
- bindings give you inline code checking in IDEs

especially point two is something really important IMHO. If I use a binding here an have a typo in `bold`, the IDE tells me, that I'm using a wrong type here. That is a great advantage in TS, and I'd rather not miss it ;)